ls1intum/Artemis
# [Enhancement] Add bonus to views and some renamings

### Checklist

### Motivation and Context


As we added bonus points to exercises, we want to display them as well in different views.

### Description

- Added bonus points to some views
- Changed some translations (Exercise Type -> Type, Max Points -> Points)

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration and 
3. Go to Assessment/Instructor Dashboard of a course and then to Exercise Dashboard 
4. Check if for exercises with bonus points the bonus points are displayed as well also (Exercise Type -> Type, Max Points -> Points)
5. Navigate to Course Overview and go into a course
6. Enter a Exercise and check if for exercises with bonus points the bonus points are displayed as well
7. Go to Course Administration and then to Exams
8. Check Exercise Groups and check that it says Points and not Max. Points
9. Participate in Exam (or do a test run) check if for exercises (modeling, text and programming) with bonus points the bonus points are displayed as well

Check also if I missed some other spots where this can be changed/added (see also comments in discussion). Just let me know :) 
### Test Coverage

Nothing changed

### Screenshots



Student Exercise Details View:
Bonus:
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/41108487/97758822-02736900-1b00-11eb-817a-3b0c8572b6a5.png">

No Bonus:
<img width="1676" alt="image" src="https://user-images.githubusercontent.com/41108487/97758861-174ffc80-1b00-11eb-80a9-ab62fd0c3091.png">

Assessment Dashboard:
<img width="1675" alt="image" src="https://user-images.githubusercontent.com/41108487/97758899-29319f80-1b00-11eb-9fc3-a2987d9123fc.png">

Instructor Dashboard:
<img width="1679" alt="image" src="https://user-images.githubusercontent.com/41108487/97758926-35b5f800-1b00-11eb-91df-0f8f2cd3388e.png">

Exam:
<img width="1399" alt="image" src="https://user-images.githubusercontent.com/41108487/97758990-60a04c00-1b00-11eb-8d24-5e467dc092be.png">

<img width="1654" alt="image" src="https://user-images.githubusercontent.com/41108487/97759048-7f9ede00-1b00-11eb-9fa1-c09e935a191b.png">
